### ISSUE TITLE
Mulitple upgrades to training scripts

### ISSUE THREAD

**Issue by dufaultc** [McMasterAI/Radiology-and-AI#47]
- add learning rate parameter to lightning module
- Add wandb logger
- Try to determin how to add hyperparameter search with pytorch lightning
- Save pre-created eval and train dataset
- Update metric logging
